db/schema.rb

```diff
@@ -86,7 +86,7 @@
     t.string "to_state", null: false
     t.text "metadata"
     t.integer "sort_key", null: false
-    t.bigint "solr_document_sidecar_id", null: false
+    t.integer "solr_document_sidecar_id", null: false
```

Comment:
PG should speak bigint... was this change really necessary?

Reply:
Indeed pg does know bigint, and the databases were created with bigint. I don't know what rails did here with this in schema.rb (I never know what rails does in schema.rb). I'll pull this out.